syntax = "proto3";

option java_package = "com.bookmyshow.proto";
option java_outer_classname = "EventProto";

message Event {
  string eventId = 1;
  string title = 2;
  string description = 3;
  string eventType = 4;
  string categoryType = 5;
  string organizerName = 6;
  string startDate = 7;
  string endDate = 8;
  string bannerUrl = 9;
}

message EventInput {
  string title = 1;
  string description = 2;
  string eventType = 3;
  string categoryType = 4;
  string organizerName = 5;
  string startDate = 6;
  string endDate = 7;
  string bannerUrl = 8;
}

message EventsList {
  repeated Event events = 1;
}
